code-challenges.test.js

```diff
@@ -14,56 +15,170 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const num1 = 15
+//const num1 = 15
 // Expected output: "15 is divisible by three"
-const num2 = 0
+//const num2 = 0
 // Expected output: "0 is divisible by three"
-const num3 = -7
+//const num3 = -7
 // Expected output: "-7 is not divisible by three"
 
+describe('divisibleBy3', () => {
+    it('take a number as an argument and decides if the number is evenly divisible by three or not', () => {
+        const num1 = 15
+        // Expected output: "15 is divisible by three"
+        const num2 = 0
+        // Expected output: "0 is divisible by three"
+        const num3 = -7
+        // Expected output: "-7 is not divisible by three"
+
+        expect(divisibleBy3(num1)).toEqual("15 is divisible by three")
+        expect(divisibleBy3(num2)).toEqual("0 is divisible by three")
+        expect(divisibleBy3(num3)).toEqual("-7 is not divisible by three")
+    })
+})
+
+// Good failure
+
+// PSEUDO CODE for a & b:
+// create a function named divisibleBy3
+// divisibleBy3 will have a parameter of number - which will be a number
+// will check if that number is divisible by three or not
+// will return a string with the number with the remarks "is divisble by three", " is not divisible by three"
+
+
 
 // b) Create the function that makes the test pass.
 
+// will check if that number is divisible by three or not
+// will return a string with the number with the remarks "is divisble by three", " is not divisible by three"
+
+const divisibleBy3 = number => number % 3 === 0 ? `${number} is divisible by three` : `${number} is not divisible by three`
+
+// Test Passed
 
 
 // --------------------2) Create a function that takes in an array of words and returns an array with all the words capitalized.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+// const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
-const randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
+// const randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
 
+describe('capitalizedWords', () => {
+    it('take in an array of words and returns an array with all the words capitalized', () => {
+        const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
+        // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+        const randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
+        // Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
```

Comment:
In the essence of your clean code techniques (which are superb), I suggest using a variable to hold the expected output. Then pass that variable into your expect statements.